cas-server-core-authentication/src/main/java/org/jasig/cas/authentication/DefaultAuthenticationTransactionManager.java

```diff
@@ -15,13 +13,11 @@
  * @since 4.2.0
  */
 @Component("defaultAuthenticationTransactionManager")
-public final class DefaultAuthenticationTransactionManager implements AuthenticationTransactionManager {
+public class DefaultAuthenticationTransactionManager implements AuthenticationTransactionManager {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(DefaultAuthenticationTransactionManager.class);
 
-    @NotNull
-    @Autowired
-    @Qualifier("authenticationManager")
+    @Resource(name="authenticationManager")
```

Comment:
For consistency and also `@Resource` did not work for me in one scenarios (don't remember which. Could be constructor injection, etc.). In general `@Autowired` is more powerful and leverages the full power of Spring, etc. Let's pick a autowiring injection style and stick to it.